apps/test/unit/code-studio/pd/form_components_func/FormControllerTest.js

```diff
@@ -180,6 +180,7 @@ describe('FormController', () => {
     });
 
     describe('Saving', () => {
+      // [MEG] TODO: Refactor this to use savedStatus instead of form_data
```

Comment:
I'm leaving this as a to-do because soon, we'll need to have a 'reopened' status if an RP reopens an application. We should check savedStatus for both of these cases.